site/content/posts/2023-09-18-multiple-idps-and-identity-transformations.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Pinniped v0.26.0: Multiple identity providers and identity transformations"
+slug: multiple-idps-and-identity-transformations
+date: 2023-09-18
+author: Ryan Richard
+image: https://plus.unsplash.com/premium_photo-1661962912663-49c457dda9ca?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=4144&q=80
+excerpt: "With the release of v0.26.0, Pinniped now supports multiple identity providers and identity transformations"
+tags: ['Ryan Richard', 'release']
+---
+
+![multiple seals](https://plus.unsplash.com/premium_photo-1661962912663-49c457dda9ca?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=4144&q=80)
+*Photo from [Unsplash](https://unsplash.com/photos/fcmxfsAgxqU)*
+
+Pinniped's v0.26.0 release includes several significant new features described below.
+Additionally, the release includes several dependency updates and fixes.
+See the [release notes](https://github.com/vmware-tanzu/pinniped/releases/tag/v0.26.0) for more details.
+
+## Background: Identity provider resources
```

Comment:
It would be nice to convey the idea:
```
Before: One Identity Provider for Many Clusters
Then: Many Identity Providers for Many Clusters
Now: Many Identity Providers Grouped in Many Ways for Many Clusters!
```
I realize the "then" and "now" all came in one chunk, but this is a pretty bit improvement of the configuration matrix.  Not only one to many IDPs, but one to many collections of many IDPs for many clusters.  😅 🥵